carla-carma-integration/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+FROM ubuntu:16.04
+FROM ros:kinetic
+WORKDIR /home
+
+# CARLA PythonAPI
+RUN mkdir ./PythonAPI
+ADD https://carla-releases.s3.eu-west-3.amazonaws.com/Backup/carla-0.9.10-py2.7-linux-x86_64.egg ./PythonAPI
```

Comment:
How likely is it that this URL breaks in the future or the contents it points at to change? If it's possibly volatile, it might be worth setting up a base image for simulation in the future (as a new user story) to store all of these dependencies in one place.